src/HospitalAPI/Controllers/Intranet/RoomsController.cs

```diff
@@ -42,6 +42,15 @@ public IActionResult UpdateName(int id, [FromBody] string name)
             return Ok(room);
         }
 
+        [Route("equipmentView/{roomId:int}")]
```

Comment:
If you are getting equipment by room, you should prefix route with room. So instead of equipmentView/{roomId} you should write room/{roomId}, so that the final route becomes api/intranet/equipment/room/1 for example